backend/utils/constants.py

```diff
@@ -15,6 +15,12 @@ class Account:
     ORGANIZATION_ID = "organization_id"
 
 
+class FeatureFlag:
+    """Temporary feature flags."""
+
+    REMOTE_FILE_STORAGE = "remote_file_storage"
+
+
```

Comment:
@harini-venkataraman please use the class in `backend/backend` instead
cc: @muhammad-ali-e